### PR TITLE
Fix: avoid warning

### PR DESCRIPTION
**Pull request by hregis**
Declaration of doc_generic_user_odt::get_substitutionarray_object($object, $outputlangs) should be compatible with CommonDocGenerator::get_substitutionarray_object($object, $outputlangs, $array_key = 'object')

### CODE REVIEW

File: htdocs/core/modules/user/doc/doc_generic_user_odt.modules.php [hunk 1]
```diff
@@ -420,12 +420,12 @@ function write_file($object,$outputlangs,$srctemplatepath,$hidedetails=0,$hidede
 		return -1;
 	}
 
-	function get_substitutionarray_object($object,$outputlangs) {
+	function get_substitutionarray_object($object,$outputlangs,$array_key='object') {
 		foreach($object as $key => $value) {
-			if(!is_array($value) && !is_object($value)) {
-	    	$array_other['object_'.$key] = $value;
+			if (!is_array($value) && !is_object($value)) {
+				$array_other['object_'.$key] = $value;
```
Comment (eldy) [inline htdocs/core/modules/user/doc/doc_generic_user_odt.modules.php]: So here we should have 
$array_other[$array_key.'_'.$key] = $value;
?

Comment (hregis) [reply htdocs/core/modules/user/doc/doc_generic_user_odt.modules.php]: @eldy 
i don't know, i just add the missing parameter for avoid warning ! :-)

Comment (hregis) [reply htdocs/core/modules/user/doc/doc_generic_user_odt.modules.php]: @eldy fixed

**Comment by hregis** [comment]
@eldy fixed

**Pull request merged**